### PR TITLE
Path - Update Enum & Packet Classes

### PR DESCRIPTION
**Pull request by kirmmin**
Updated Player.cs to expose ServerPathLog properties

### CODE REVIEW

**Comment by kirmmin** [comment]
The following commit contains further updates to this: https://github.com/lucas-jones/NexusForever/commit/93833e0149c7de14b177ec8c8fe19b599262fe9f

File: Source/NexusForever.WorldServer/Network/Message/Model/ServerPathLog.cs [hunk 1]
```diff
@@ -1,13 +1,32 @@
...
+        * Settler, Scientist = 6
+        * Soldier, Scientist, Settler = 7
+        * Explorer, Soldier = 9
+        * Explorer, Settler = 10
+        * Explorer, Soldier, Settler = 11
+        * Explorer, Scientist = 12
+        * Explorer, Soldier, Scientist = 13
+        * Explorer, Settler, Scientist = 14
+        * All = 15
+        **/
```
Comment (Rawaho) [inline Source/NexusForever.WorldServer/Network/Message/Model/ServerPathLog.cs]: This doesn't need to be a comment.
Either create an enum for this or use the existing path enum and just bitshift it 1 << path which will result in the same thing.

File: Source/NexusForever.WorldServer/Game/Entity/Static/Path.cs [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
     public enum Path
     {
-        Soldier,
-        Settler,
-        Scientist,
-        Explorer,
+        Soldier = 0,
+        Settler = 1,
+        Scientist = 2,
+        Explorer = 3,
```
Comment (Rawaho) [inline Source/NexusForever.WorldServer/Game/Entity/Static/Path.cs]: To follow with the rest of the project style, you only need to explicitly give the values when the enum isn't sequential or doesn't start from 0

File: Source/NexusForever.WorldServer/Game/Entity/Player.cs [hunk 3]
```diff
@@ -21,6 +22,9 @@ namespace NexusForever.WorldServer.Game.Entity
 {
     public class Player : WorldEntity, ISaveCharacter
     {
+
```
Comment (Rawaho) [inline Source/NexusForever.WorldServer/Game/Entity/Player.cs]: Extra space doesn't need to be there

**Review by Rawaho — CHANGES_REQUESTED** [review]

**Comment by kirmmin** [comment]
This is complete and ready for merge. This sets us up to bring in the work currently happening over on https://github.com/kirmmin/NexusForever/tree/player-path-management. This branch, as of this posting, enables path data upon character creation & defaults to soldier if no path data available upon logging in.
There will be some updates to path switching and /who results in relation to this, too.

Big thanks to @lucas-jones for all the help with path opcodes as we build out this piece.

**Comment by kirmmin** [comment]
Replaced by #74

**Pull request closed without merging**